pom.xml

```diff
@@ -139,13 +139,23 @@
             <modules>
                 <module>tests-spark310+</module>
             </modules>
+            <repositories>
+                <!--
+                TODO remove once Apache Spark 3.1.1 is released.
+                Last 3.1.1-SNAPSHOT is before SPARK-34223
```

Comment:
Is this an issue with the public snapshots or the snapshots from URM?  If it was passing for you locally using the Apache snapshots but failing in CI then it's likely an issue with URM snapshots and we should just fix the URM snapshot build.

Reply:
it's the "issue" with public spark snapshots